ground/gcs/src/plugins/coreplugin/generalsettings.cpp

```diff
@@ -189,10 +189,14 @@ void GeneralSettings::readSettings(QSettings* qs)
     m_proxyHostname = qs->value(QLatin1String("proxyhostname"),m_proxyHostname).toString();
     m_proxyUser = qs->value(QLatin1String("proxyuser"),m_proxyUser).toString();
     m_proxyPassword = qs->value(QLatin1String("proxypassword"),m_proxyPassword).toString();
-    m_forumUser = qs->value(QLatin1String("forumuser"),m_forumUser).toString();
-    m_forumPassword = qs->value(QLatin1String("forumpassword"),m_forumPassword).toString();
-    m_aircraft = qs->value(QLatin1String("aircraft"),m_aircraft).toString();
-    m_observations = qs->value(QLatin1String("observations"),m_observations).toString();
+    m_observations = qs->value(QLatin1String("observations"), "").toString();
+    m_vehicle = qs->value(QLatin1String("vehicle"), "").toString();
+    m_board = qs->value(QLatin1String("board"), "").toString();
+    m_weight = qs->value(QLatin1String("weight"), 0).toInt();
```

Comment:
I have a feeling I entered '225mtm' in here and it turned it into '0', if that's intended.. might be better to string, then parse/atoi/trim..